build.rs

```diff
@@ -181,6 +184,13 @@ fn main() -> Result<(), Box<dyn Error>> {
     }
     bundler.finish().expect("failed to finish bundle");
 
+    let car_contents = std::fs::read(&dst).expect("failed to read CAR file");
+    let mut encoder = GzEncoder::new(Vec::new(), Compression::best());
+    encoder.write_all(&car_contents).expect("failed to compress CAR");
+    let compressed = encoder.finish().expect("failed to finish compression");
+
+    std::fs::write(&dst, compressed).expect("failed to write compressed bundle");
+
```

Comment:
We expect clients to compress these car files (e.g., see lotus where we compress with zstd).